src/components/allproperties/PropertiesInfo.jsx

```diff
@@ -54,13 +51,11 @@ function PropertiesInfo(props){
                   </span>
```

Comment:
I suggest you add an alt text on your image incase the image fail to load

Reply:
> I suggest you add an alt text on your image incase the image fail to load

okay.